btp.lua

```diff
@@ -0,0 +1,486 @@
+                                        nil)
+local account_id = ProtoField.string("btp.account_id", "Account ID", base.ASCII,
+                                     nil)
+local cti_type = ProtoField.uint8("btp.cti_type", "CTI Type", base.DEC, nil,
+                                  nil, nil)
+local firm_name_len = ProtoField.uint8("btp.firm_name_len", "Firm Name Length",
+                                       base.DEC, nil, nil, nil)
+local firm_name = ProtoField.string("btp.firm_name", "Firm Name", base.ASCII,
+                                    nil)
+local firm_type = ProtoField.uint8("btp.firm_type", "Firm Type", base.HEX,
+                                   firm_type_string, nil, nil)
+local user_memo_len = ProtoField.uint8("btp.user_memo_len", "User Memo Length",
+                                       base.DEC, nil, nil, nil)
+local user_memo = ProtoField.string("btp.user_memo", "User Memo", base.ASCII,
+                                    nil)
+local modify_id = ProtoField.uint64("btp.modify_id", "Modify ID", base.DEC, nil,
+                                    nil, nil)
+local old_price = ProtoField.int32("btp.old_price", "Old Price", base.DEC, nil,
+                                   nil, nil)
+local old_quantity = ProtoField.uint32("btp.old_quantity", "Old Quantity",
+                                       base.DEC, nil, nil, nil)
+local filled_quantity = ProtoField.uint32("btp.filled_quantity",
+                                          "Filled Quantity", base.DEC, nil, nil,
+                                          nil)
+local liquidity = ProtoField.uint8("btp.liquidity", "Liquidity", base.HEX,
+                                   liquidity_string, nil, nil)
+local close_reason = ProtoField.uint8("btp.close_reason", "Close Reason",
+                                      base.HEX, close_reason_string, nil, nil,
+                                      nil)
+local order_entry_reject_reason = ProtoField.uint8(
+                                      "btp.order_entry.reject_reason",
+                                      "Reject Reason", base.HEX,
+                                      order_entry_reject_reason_string, nil, nil)
+local time_in_force = ProtoField.uint8("btp.time_in_force", "Time in Force",
+                                       base.HEX, time_in_force_string, nil, nil)
+local order_entry_message_type = ProtoField.uint8(
+                                     "btp.order_entry.message_type",
+                                     "Message Type", base.HEX,
+                                     order_entry_message_type_string, nil, nil)
+local drop_copy_message_type = ProtoField.uint8("btp.drop_copy.message_type",
+                                                "Message Type", base.HEX,
+                                                drop_copy_message_type_string,
+                                                nil, nil)
+
+local price_feed_message_type = ProtoField.uint8("btp.price_feed.message_type",
```

Comment:
nit: typically we have `pricefeed` as one word.

Reply:
Fixed